src/components/core/DynamicImage/DynamicImage.tsx

```diff
@@ -10,26 +10,32 @@ import {
   ImageSourcePropType,
   ImageStyle,
   ImageURISource,
+  LayoutChangeEvent,
   StyleProp,
   StyleSheet,
-  View
+  View,
+  ViewStyle
 } from 'react-native'
 
 import { ImageSkeleton } from 'app/components/image-skeleton'
 
 export type DynamicImageProps = {
   // Image source
-  image?: ImageSourcePropType
-  // Style to apply to the image itself
-  style?: StyleProp<ImageStyle>
+  source?: ImageSourcePropType
+  styles?: {
+    root?: ViewStyle
```

Comment:
Can update these to allow for array of styles
```suggestion
    root?: StyleProp<ViewStyle>
```

Reply:
ah yeah great call, is that what StyleProp does basically?

Reply:
Yea that's my understanding. Sebastian mentioned it before and I believe him haha